src/app/cube/home/learning-object-info/learning-outcomes/learning-outcomes.component.scss

```
@@ -1,15 +1,78 @@
-.content{
+@import 'vars.scss';
+
+.content {
     margin-top: 50px;
     margin-right: 50px;
+    padding-left: 0px;
     display: flex;
     flex-direction: column;
-    .taxonomy{
-      margin-bottom: 400px;
+    
+    .blooms-cards {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+        gap: 25px;
```

```suggestion
        gap: 50px;
```

To match the gap between the cards in collection